modules/engage-theodul-plugin-tab-downloads/src/main/java/org/opencastproject/engage/theodul/plugin/tab/downloads/EngagePluginTabDownloads.java

```diff
@@ -53,8 +53,8 @@ protected void activate(ComponentContext cc) {
   }
 
   @Reference
-  protected void setPluginManager(EngagePluginManager pm) {
-    super.setPluginManager(pm);
+  protected void setPluginManager(final EngagePluginManager pluginManager) {
```

Comment:
* Why add the `final` only here?
* Why not adding `@Override` only here?

Reply:
fixed